README.md

```diff
@@ -22,17 +22,18 @@
 # PROPOSAL [![Build Status](https://travis-ci.org/tudo-astroparticlephysics/PROPOSAL.svg?branch=master)](https://travis-ci.org/tudo-astroparticlephysics/PROPOSAL) [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3665606.svg)](https://doi.org/10.5281/zenodo.3665606)
 
 PROPOSAL (Propagator with optimal precision and optimized speed for all
-leptons) is presented as a public tool for muon propagation through transparent
-media. Up-to-date cross sections for ionization, bremsstrahlung, photonuclear
+leptons) is presented as a public tool for propagating leptons and gamma rays
```

Comment:
gamma photons

Reply:
gamma photons? Never heard that expression, it's either he photons, gammas or gamma rays